core/logger/slog.go

```diff
@@ -0,0 +1,156 @@
+package logger
```

Comment:
This file is inefficient but proves that it's possible to partition logs based on the context.